packages/components/src/SelectList/SelectListItem.tsx

```diff
@@ -1,7 +1,9 @@
-import { ReactNode, forwardRef } from 'react';
+import type { ReactNode } from 'react';
+import { forwardRef } from 'react';
 import type RAC from 'react-aria-components';
-import { GridListItem as SelectListItem } from 'react-aria-components';
-import { SVGProps, cn } from '@marigold/system';
+import { GridListItem as RACGridListItem } from 'react-aria-components';
```

Comment:
I think like it was called before was also fine: `import { GridListItem as SelectListItem } from 'react-aria-components';`